ci/audit.yml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-nancy
+    tag: latest
+
+inputs:
+  - name: dp-frontend-renderer
+    path: dp-frontend-renderer
```

Comment:
Something mentioned to me for one of my PRs on this task is that we don't need the `inputs.path` value anymore, so may be worth removing